<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddList</class>
 <widget class="QMainWindow" name="AddList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add List</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-image: url(:/images/background/backverticalback.png);
 }
QGroupBox {
border: 2px solid #aa55ff;
border-radius: 15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>85</y>
      <width>250</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 15pt &quot;Merriweather&quot;;
</string>
    </property>
    <property name="title">
     <string>Add List</string>
    </property>
    <widget class="QLabel" name="labelColor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Merriweather&quot;;</string>
     </property>
     <property name="text">
      <string>Color :</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelListName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>38</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Merriweather&quot;;</string>
     </property>
     <property name="text">
      <string>List Name :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditListName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
font: 9pt&quot;Merriweather&quot;;
border-radius: 7px;
background-color: rgb(215, 200, 255);
border: 1px solid #000000;
padding: 5px;
}
QLineEdit:hover {
	background-color: rgb(206, 192, 247);
}</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelError">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>250</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Merriweather&quot;;
color: rgb(255, 0, 0);
qproperty-alignment: 'AlignHCenter';</string>
     </property>
     <property name="text">
      <string>error</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelBackground">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/background/small_background_blue.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBlue">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/icon/blue_color.png</normaloff>
       <normalon>:/images/icon/select_blue_color.png</normalon>:/images/icon/blue_color.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonGreen">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/icon/green_color.png</normaloff>
       <normalon>:/images/icon/select_green_color.png</normalon>:/images/icon/green_color.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPink">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/icon/pink_color.png</normaloff>
       <normalon>:/images/icon/select_pink_color.png</normalon>:/images/icon/pink_color.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPurple">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/icon/purple_color.png</normaloff>
       <normalon>:/images/icon/select_purple_color.png</normalon>:/images/icon/purple_color.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>315</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover {
  background-color: rgb(165, 125, 235);
}
QPushButton {
	font: 10pt&quot;Merriweather&quot;;
	border-radius: 7px;
	background-color: rgb(180, 135, 255);
	border: 1px solid #000000;
	color: rgb(0, 0, 0);
}
QPushButton:pressed {
	background-color: rgb(135, 100, 195);
}</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
